contents/blog/making-something-people-want.md

```diff
@@ -0,0 +1,149 @@
+
+All messages were totally cutomized to the person. Nothing automated. I was very clear I was just hoping that we could get them as an early, free user - because we were preparing the product before a big launch.
+
+Within a couple of days I had a few teams who were up for trying it.
+
+We spent a further $2K on ads on Twitter to see if we could get a little self serve usage up and running - our main focus here was to (i) check nothing was broken, in case it went viral when we launched and (ii) get the repo to 100 stars. I'd stress we had a hard 4 week deadline we'd set ourselves, so we were spending money to avoid spending more time to let the above happen naturally. This step wasn't necessary, so please don't copy if you've no capital.
+
+2. Give user access to new feature
+
+Our only focus early on, was to make sure the product did the bare minimum. The first ever team could (i) login (well, after we fixed the login screen - that was the first bug) and (ii) see a list of events streaming from their website.
+
+The reason they were willing to let us implement something with so little value, at that point (!), was that they were friends with us and perhaps believed in the vision!
+
+3. User uses it and gives feedback
+
+We set up a [community Slack](../slack) group.
+
+With each customer that would join, we'd create a private channel. In the early days when we were a team of two, this is where users went from my responsibility, to Tim's.
+
+This made us feel very close with users.
+
+We realized we had to _act_ on feedback/bugs and feature requests. We knew that if we acted on them, we'd get more, which means we'd make more progress. If we didn't do this, then users would stop giving us feedback and we'd die!
+
+4. Repeat
+
+For every new feature, we aimed to get a few early adopter users on it, and through the above loop. Every time we did so, this led to bug reports, UX improvements or something totally different. A bit like QA-ing a pull request.
+
+It is _much _ better to ship like this, rather than shipping in isolation. With this approach, what you build will be much more likely to work, and you'll gain a far deeper understanding of your customers too.
+
+This _still_ applies. See our [community roadmap idea](https://github.com/PostHog/posthog.com/issues/4453) - we're looking to go further with this.
+
+## How to engage with users
+
+Once we launched, we made it _very_ easy to give us feedback. Slack turned out to be the best channel because:
+
+* many users had it open all day at work - we saw 20x the response rate of email
+* there's a low bar to posting because people use it informally - people don't spend a lot of time to polish their communication on it. We wanted more communication, rather than less, especially in the early days
+
+There are some issues with using Slack like this:
+
+* It isn't ticketed (without looking for 3rd party apps) - this can be painful now we've several thousand people in there
```

Comment:
```suggestion
* It can be difficult to triage issues because it isn't ticketed (without looking for 3rd party apps) - this can be painful now we've several thousand people in there
```

Reply:
I want to keep the 'this assumes I'm a developer and will already get this' vibe my original version suggests